deeptools/_version.py

```diff
@@ -2,4 +2,4 @@
 # This file is originally generated from Git information by running 'setup.py
 # version'. Distribution tarballs contain a pre-generated copy of this file.
 
-__version__ = '1.6.0'
+__version__ = '1.6.0-173-gc811461'
```

Comment:
I would hope not. Truth be told I think it's going to be `__version__ = '2.0.0'` anyway.